requirements.txt

```diff
@@ -75,7 +75,7 @@ pydantic==1.7.3
 pyflakes==2.2.0
 pyparsing==2.4.7
 pytest==6.2.1
-pytest-asyncio<0.14.0
+pytest-asyncio==0.12.0
```

Comment:
Why?

Reply:
i have no idea where it came from :) - Fixed.